R/pkg/R/serialize.R

```diff
@@ -83,6 +83,7 @@ writeObject <- function(con, object, writeType = TRUE) {
          Date = writeDate(con, object),
          POSIXlt = writeTime(con, object),
          POSIXct = writeTime(con, object),
+         bigint = writeDouble(con, object),
```

Comment:
if you are referring to https://github.com/apache/spark/blob/master/R/pkg/R/types.R#L25
like it says, `names(PRIMITIVE_TYPES) are Scala types whereas
# values are equivalent R types.` so `bigint` there is Scala type, not R type